docs/Training-on-Amazon-Web-Service.md

```diff
@@ -7,10 +7,10 @@ Service for training ML-Agents environments.
 
 We've prepared an preconfigured AMI for you with the ID: `ami-18642967` in the
 `us-east-1` region. It was created as a modification of [Deep Learning AMI
-(Ubuntu)](https://aws.amazon.com/marketplace/pp/B077GCH38C). If you want to do
-training without the headless mode, you need to enable X Server on it. After
-launching your EC2 instance using the ami and ssh into it, run the following
-commands to enable it:
+(Ubuntu)](https://aws.amazon.com/marketplace/pp/B077GCH38C). We've tested it
```

Comment:
Change to:

The AMI has been tested with p2.xlarge instance. Furthermore, if you want to train without headless mode, you need to enable X Server. 

After launching your EC2 instance using the ami and ssh into it, run the following commands to enable it: